installation/docker.md

```diff
@@ -73,7 +73,7 @@ mkdir /opt/openhab
 mkdir /opt/openhab/conf
 mkdir /opt/openhab/userdata
 mkdir /opt/openhab/addons
-chown -R openhab:openhab /opt/openhab
+chown 9001.9001 /opt/openhab -R
```

Comment:
The format `chown -R openhab:openhab /opt/openhab` is more common, why the change?